source/style-guide/writing/use-active-voice.txt

```diff
@@ -4,16 +4,30 @@
 Use Active Voice
 ================
 
+.. default-domain:: mongodb
+
 .. include:: /includes/styles/corrections.rst
 
-Active voice makes the *performer* of the action (usually the user) the
-subject of the sentence. Passive voice makes the *recipient* of the
-action (not the performer) the subject of the sentence.
+.. _why-active-voice:
+
+Why Use Active Voice?
+---------------------
+
+- *Active voice* makes the *performer* of the action (usually the
+  reader or user) the subject of the sentence.
+
+- *Passive voice* makes the *recipient* of the action (not the
+  performer) the subject of the sentence.
 
-Active-voice sentences are more engaging, less complicated, less wordy,
-and easier to understand than passive-voice sentences. With active
-voice, the actions and responses of the user are easier to distinguish
-from the actions and responses of the technology.
+When compared to passive-voice sentences, active-voice sentences are:
+
+- more engaging
+- less complicated
+- less wordy
+- easier to understand
+
+With active voice, you can distinguish the actions and responses
+of the user from those of the technology.
```

Comment:
Possibly helpful add:

> This is especially important for beginner users, who are looking for specific calls to action in procedurals. A step that reads "The service can be restarted" might seem to be a consideration, or a possibility, but not an instruction. If the user fails to restart the service right here, in this step, the entire rest of the tutorial fails. Using "Restart the service" instead removes all doubt, and saves the. tutorial. Further, some actions take place automatically. A service might automatically restart itself after a failure. Using direct, active language helps to differentiate from situations where action might happen alongside, or tangentially to a user action, but not require it per se. "Restart the server" leaves no confusion over who is to perform the action here: the user.

Maybe too much? Feel free to paraphrase or disregard.

Reply:
I cleaned it up, but included it.